weasyl/controllers/api.py

```diff
@@ -152,6 +152,25 @@ def tidy_submission(submission):
     return submission
 
 
+def tidy_journal(journal):
```

Comment:
I think it’s ideal if this returns a new journal instead of modifying the old one, to help make the fields we expect to produce explicit and also to avoid a new field being accidentally exposed in the future. (It’s a bit unlikely with the state of weasyl-old, but it seems like the safe choice.)

… even though `tidy_submission` mutates. (`get_tidy_journal` might be a good distinction.)